tools/topology/m4/utils.m4

```diff
@@ -49,9 +49,9 @@ define(`COMP_FORMAT_NAME',
 	$1, `float', `FLOAT_LE',
 	)')
 
-dnl P_GRAPH(name, CONNECTIONS)
+dnl P_GRAPH(name, Pipeline_ID, CONNECTIONS)
```

Comment:
Why some names are lower case and some are uppercase, but this one is even camelcase with underscore?
I'm not m4, so I don't know if it's supposed to be like that but it looks inconsistent.

Reply:
Yeah I'd argue the macros should be uppercase and parameter names in description lowercase, for consistency with the rest of the m4 files in this project. This one is just, ugh.

Reply:
Good point, lets try and keep it uppercase atm.

Reply:
fixed now @lgirdwood